src/formpack/utils/flatten_content.py

```diff
@@ -151,23 +156,24 @@ def _flatten_translated_fields(row, translations, translated_cols,
 
     _placed_cols = set()
 
-    def _place_col_in_order(col, base_col=None):
```

Comment:
Not really, but in the same method, we use another variable calls `col` as well. 
TBH, I was debugging something and wanted to be sure `_place_col_in_order()` was receiving the correct value. Because, it didn't hurt, I left it as is. 
I can put it back if you want.